### ISSUE TITLE
Message pane is white when no message is selected

### ISSUE THREAD

**Issue by rafaelhlima** [fastaddons/Darko#18]
**Describe the bug**  
In Thunderbird 91.5 when you open the application, no message is selected, so the message pane is white (see attached screenshot).

![Darko](https://user-images.githubusercontent.com/65458584/154071429-e72d871d-060b-4c2c-b578-7e97aa2af6b7.png)

I believe this color should be themed using a dark color as well.

BTW. Thanks for this add-on!!

**Steps To Reproduce**  
1. Install Darko
2. Open Thuderbird with no selected message.
3. The message pane is white.

**Expected behavior**  
The message pane should have a dark color.

**Device info:**
 - operating system: Kubuntu 21.10 (KDE 5.24)
 - Browser and version: Thunderbird 91.5
 - addon version: Darko 3.3

**Comment by fastaddons** [comment]
Sooooo.... I've been playing with it for more than an hour now but I'm unable to fix it. Although I'm able to change the color but it seems I'm unable to detect that user clicked a folder where no message is displayed without additional permission.

BUT there is a good news, in the new Thunderbird Alpha 99 it works because the color is taken from the theme, so when you use dark theme, it will be dark. So I'm pretty sure this will be fixed with a Thunderbird update, just don't know which one.
I wish I could help more, but I'm super busy right now with my other addon :(

**Comment by fastaddons** [comment]
My Thunderbird just updated to 91.6.1 (64-bit) the empty message pane is now dark. So I think it's fixed now.

**Comment by rafaelhlima** [comment]
> My Thunderbird just updated to 91.6.1 (64-bit) the empty message pane is now dark. So I think it's fixed now.

I'm waiting Thunderbird to update on my distro's repository. I'll let you know if it worked when it updates to 91.6.

**Comment by falkenhawk** [comment]
@fastaddons you are right, this is fixed now in thunderbird, message panel is dark when no message is selected.
But, the message panel often flashes white for a fraction of a second when you switch between messages. Could those flashes be eliminated somehow?

**Comment by fastaddons** [comment]
Not from my addon, sadly the API doesn't support injecting CSS before the message is fully loaded (unlike in browsers where one can use `runAt: 'document_start'`):
https://webextension-api.thunderbird.net/en/latest/messageDisplayScripts.html#messagedisplayscripts

**Comment by falkenhawk** [comment]
Ok, no problem.
I worked around it by setting just the background color to #2a2a2e (r 42 g 42 b 46), while leaving the font color at #000000, because it gets inverted to white in the end by your extension.

![image](https://user-images.githubusercontent.com/14310995/194508167-304f6c8e-d12e-4161-a18b-c3f5e326123f.png)

**Comment by mherrmann3** [comment]
@falkenhawk Thanks! But then it briefly flashes black when using the light mode...
(I often switch between both themes depending on the ambient light situation)

As a compromise, I'll use 127,127,127 😉

**Edit**: I found a better solution on my quest to switch the message theme automatically with the OS theme, in which I also found a solution to fixes this issue, see https://github.com/fastaddons/Darko/issues/8#issuecomment-1327748181

**Comment by mherrmann3** [comment]
Y'all may be interested in my alternative message-darkening solution: https://github.com/fastaddons/Darko/issues/8#issuecomment-1327748181, which is not affected by any white/dark/gray flashes (nor white message pane when no message is selected).

(Actually, my main motivation was to automatically switch the message theme with the OS theme. Surprisingly, I could solve all that simply with a `userChrome.css` & `userContent.css`. For now, I don't need Darko anymore 😢, unless it implements all those things, which gives the additional benefit of quickly switching the message theme manually if need be.)
